README.md

```diff
@@ -47,6 +47,7 @@ In future Crypto may also be included and emphasis will be given on improving th
 
 ### Steps to Contribute
 * Fork the Repo
+* Clone the forked repo
```

Comment:
Also, add to make a new branch the steps should be as follows:

1. Fork the repo
2. Clone the forked repo
3. Checkout from the master branch to a new branch.
4. Take an issue
5. Commit changes to the new branch for every issue 
6. Commit the changes to the new branch
7. Make a PR request from the new branch

Reply:
May be something like this is even better
1. Fork the repo
2. Clone the forked repo
3. Take an issue
4. Checkout to a new branch for every issue
5. Commit the changes to the new branch and squash them to be one in right formar.
6. Commit message format is **Issue `#ISSUE_NO: SINGLE LINE DISCRIPTION OF ISSUE**
   Eg: Issue #1: Update README.md to include project details
7. Make a PR request from the new branch

Reply:
Done

Reply:
Ayush please see the new list by me.

Reply:
@Ayush-Shukla02 ping

Reply:
is the example also needed in point 6?

Reply:
Yup, it would make things clear for everyone. And you can use this exact commit message for this PR.

Reply:
Done

Reply:
Great nice work. Just waiting for squashing of commits. Reset is done.

Reply:
Yes I have cloned it on my laptop. I will see how to do it.

Reply:
Just working under the guidance of you all. 😊